aregion.cpp

```diff
@@ -3734,13 +3737,13 @@ void economy(ARegionArray* arr, const int w, const int h) {
 			bool addLair = getrandom(100) < terrain->lairChance;
 
 			reg->ManualSetup({
-				terrain: terrain,
-				habitat: terrain->pop + 1,
-				prodWeight: 1,
-				addLair: addLair,
-				addSettlement: false,
-				settlementName: std::string(),
-				settlementSize: 0
+				.terrain = terrain,
+				.habitat = terrain->pop + 1,
+				.prodWeight = 1,
+				.addLair = addLair,
+				.addSettlement = false,
+				.settlementName = std::string(),
+				.settlementSize = 0
```

Comment:
all of these are due to CLang complaining about new-style object initialization (what I change it to) versus and older gnu-style initialization (what was there).